src/application.rs

```diff
@@ -3,7 +3,7 @@ use std::error::Error;
 
 use super::routes;
 use super::storage::{SqlStorage, Storage};
-use super::web::{RequestIdHeader, RequestSpan};
+use super::web::{JwtValidator, RequestIdHeader, RequestSpan, Validator};
```

Comment:
Sounds good! I thought that would be implied because the trait itself is in the `auth` module, but here we import it from `web`, so I see how this can be ambiguous.